easybuild/easyconfigs/p/Python/Python-3.7.4-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,321 @@
+name = 'Python'
+version = "3.7.4"
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']
```

Comment:
I think we could go with http**s**;
` source_urls = ['https://www.%(namelower)s.org/ftp/%(namelower)s/%(version)s/']`